nextflow.config

```diff
@@ -43,12 +43,12 @@ params {
     adapter_fasta              = []
 
     // Compression
-    cram                       = []
+    cram                       = false
 
     // Alignment options
     star_ignore_sjdbgtf                  = false
-    seq_center                           = null
-    seq_platform                         = null
+    seq_center                           = ""
```

Comment:
I think these values should be meta values so they can be set per sample

Reply:
Good idea! This might be out of scope for this PR though, can you open an issue for this?